eq-author/src/App/settings/ThemesPage.test.js

```diff
@@ -363,4 +363,50 @@ describe("Themes page", () => {
       })
     );
   });
+
+  it("Should display Form Type", () => {
+    renderThemesPage(mockQuestionnaire, user, mocks);
+
+    expect(screen.getByText("Form type")).toBeVisible();
+  });
+
+  it("Should handle Form Type update", () => {
+    const handleFormTypeBlur = jest.fn();
+    useMutation.mockImplementation(() => [handleFormTypeBlur]);
+    renderThemesPage(mockQuestionnaire, user, mocks);
+
+    const formTypeInput = screen.getByTestId("default-form-type-input");
+
+    fireEvent.change(formTypeInput, {
+      target: { value: "123" },
+    });
+
+    fireEvent.blur(formTypeInput);
+    expect(handleFormTypeBlur).toHaveBeenCalledWith(
+      expect.objectContaining({
+        variables: {
+          input: {
+            questionnaireId: expect.any(String),
+            formType: "123",
+            shortName: "default",
+          },
+        },
+      })
+    );
+  });
+
+  it("Should handle Form Type change when not empty", () => {
+    const handleFormTypeBlur = jest.fn();
+    useMutation.mockImplementation(() => [handleFormTypeBlur]);
+    renderThemesPage(mockQuestionnaire, user, mocks);
+
+    const formTypeInput = screen.getByTestId("default-form-type-input");
+
+    fireEvent.change(formTypeInput, {
+      target: { value: " " },
+    });
+
+    fireEvent.blur(formTypeInput);
+    expect(handleFormTypeBlur).not.toHaveBeenCalled();
+  });
```

Comment:
You could fix the failing test by modifying it to instead assert that white space is stripped out.

```suggestion
  it("Should remove spaces when there are any", () => {
    const handleFormTypeBlur = jest.fn();
    useMutation.mockImplementation(() => [handleFormTypeBlur]);
    renderThemesPage(mockQuestionnaire, user, mocks);

    const formTypeInput = screen.getByTestId("default-form-type-input");

    fireEvent.change(formTypeInput, {
      target: { value: " " },
    });

    fireEvent.blur(formTypeInput);
    
    expect(handleFormTypeBlur).toHaveBeenCalledWith(
      expect.objectContaining({
        variables: {
          input: {
            questionnaireId: expect.any(String),
            formType: "",
            shortName: "default",
          },
        },
      })
    );
  });
```